src/utils/getOptions.ts

```diff
@@ -1,6 +1,7 @@
-import RawOptions from '../types/RawOptions';
-import Options from '../types/Options';
-import normalizePath from './normalizePath';
+import RawOptions from "../types/RawOptions";
```

Comment:
Looks like I don't have prettier configured in this repo yet.  For my sanity, can you use single quotes for all the string literals in this file?

Reply:
Looks like you beat me to it!